wee_slack.py

```diff
@@ -4008,6 +4044,9 @@ class PluginConfig(object):
         'external_user_suffix': Setting(
             default='*',
             desc='The suffix appended to nicks to indicate external users.'),
+        'files_download_location': Setting(
+            default='',
+            desc='If set, file attachments will be downloaded to this location.'),
```

Comment:
I think we should say "will be automatically downloaded", to make it clear that they are downloaded without requiring any action.